CDCgov/prime-reportstream
# Experience/10387/sender bug

This PR ...

Fixes the issue when a Sender has not been defined and therefor is unable to view RS website.

Test Steps:
1. Create new account in Okta Preview and assign to group DHSender_CDC-ELIMS-HL7
2. Attempt to login to staging at https://staging.reportstream.cdc.gov/
3. See error

## Changes
- Updated the useSenderResource call to return rsUseQuery vs original method of cherry picking items 
- Added withCatchAndSuspense call when loading the BannerContent

![Screenshot 2023-07-14 at 1 40 55 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/0c7964a2-1b67-4c31-b0e3-7e21a580d3a1)

## Checklist

### Testing

## Linked Issues
- Fixes #10387
